filters/auth/basic_test.go

```diff
@@ -71,6 +72,13 @@ func TestWithSuccessfulAuth(t *testing.T) {
 	}
 }
 
+func TestWithMissingAuthFile(t *testing.T) {
```

Comment:
I think this can be added under `TestCreateFilterBasicAuthErrorCases`, but can also be done in another PR.